README.md

```diff
@@ -1,4 +1,30 @@
-# gufe
-
 [![build](https://github.com/OpenFreeEnergy/gufe/actions/workflows/ci.yaml/badge.svg)](https://github.com/OpenFreeEnergy/gufe/actions/workflows/ci.yaml)
 [![coverage](https://codecov.io/gh/OpenFreeEnergy/gufe/branch/main/graph/badge.svg)](https://codecov.io/gh/OpenFreeEnergy/gufe)
+
+# gufe - Grand Unified Free Energy
+
+GUFE is a toolkit for defining and manipulating alchemical free energy networks and calculations.
```

Comment:
This feels a bit misleading? Shouldn't we just say it's a collection of abstract classes?